connectivity/drivers/ble/FEATURE_BLE/TARGET_Ambiq_Micro/TARGET_Apollo3/CMakelists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+target_include_directories(mbed-os-ble-apollo3
```

Comment:
Following from the comment in `connectivity/drivers/ble/FEATURE_BLE/TARGET_Ambiq_Micro/CMakelists.txt`, you should create the `mbed-os-ble-apollo3` target here.